packages/@aws-cdk/aws-s3-assets/test/asset.test.ts

```diff
@@ -0,0 +1,328 @@
+import { ResourcePart, SynthUtils } from '@aws-cdk/assert';
```

Comment:
Took the liberty to convert to jest here.

Reply:
mmmmm ok... ideally not in the same PR, but that's fine

Reply:
I know... but there was only one relatively small test file in this package and it allows to use jest for the new file.